x-pack/plugins/task_manager/server/MONITORING.md

```diff
@@ -0,0 +1,242 @@
+# Task Manager Monitoring
```

Comment:
finally getting around to looking at this, sorry - I'm going to do an initial "comment" review of this doc, before looking at the code ...